guides/common/modules/proc_enabling-the-satellite-tools-repository.adoc

```diff
@@ -15,8 +15,10 @@ endif::[]
 . Use the Search field to enter the following repository name: *{project-client-name} (for RHEL 8) (RPMs)*.
 . In the Available Repositories pane, click on *{project-client-name} (for RHEL 8) (RPMs)* to expand the repository set.
 +
-If the *{project-client-name}* items are not visible, it may be because they are not included in the Red{nbsp}Hat Subscription Manifest obtained from the Customer Portal.
-To correct that, log in to the Customer Portal, add these repositories, download the Red{nbsp}Hat Subscription Manifest and import it into {Project}.
+If the *{project-client-name}* items are not visible, they must be included in the Red{nbsp}Hat Subscription Manifest obtained from the Customer Portal.
+Login to the Customer Portal, navigate to *Subscriptions* > *Subscription Allocations* and click *Red Hat Hybrid Cloud Console*.
+It redirects you to the Subscription Manifest page.
+Add these repositories, export the Red{nbsp}Hat Subscription Manifest and import it into {Project}.
```

Comment:
I suggest using just a link to the chapter [Managing Red Hat Subscriptions](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html-single/managing_content/index#Managing_Red_Hat_Subscriptions_content-management) in _Managing Content_.

If we need to update the docs regarding the Cloud Console, it should be done inside that chapter, but that is a greater task.

Reply:
Addressed :+1: